docs/getting-started/contributors.md

```diff
@@ -78,7 +75,19 @@ serves the react app ([0]) and the backend ([1]);
 Visit http://localhost:3000 and you should see the bleeding edge of Backstage
 ready for contributions!
 
-## Editor
+## Coding Guidelines
+
+All code is formatted with `prettier` using the configuration in the repo. If possible we recommend configuring your editor to format automatically, but you can also use the `yarn prettier --write <file>` command to format files.
+
+A consistent coding style is included via [EditorConfig](https://editorconfig.org/) with the file [`.editorconfig`](.editorconfig) at the root of the repo. Depending on your editor of choice, it will either support it out of the box or you can [download a plugin](https://editorconfig.org/#download) for the config to be applied.
+
+If you're contributing to the backend or CLI tooling, be mindful of cross-platform support. [This](https://shapeshed.com/writing-cross-platform-node/) blog post is a good guide of what to keep in mind when writing cross-platform NodeJS.
+
+Also be sure to skim through our [ADRs](docs/architecture-decisions) to see if they cover what you're working on. In particular [ADR006: Avoid React.FC and React.SFC](docs/architecture-decisions/adr006-avoid-react-fc.md) is one to look out for.
+
+If there are any updates in `markdown` file please make sure to run `yarn run lint:docs`. Though it is checked on `lint-staged`. It is required to install [vale](https://docs.errata.ai/vale/install) separately and make sure it is accessed by global command.
```

Comment:
```suggestion
If there are any updates in any Markdown files please make sure to run `yarn run lint:docs`. (Though it is also checked on `lint-staged` via the CI/CD pipeline.) It is required to install [Vale](https://docs.errata.ai/vale/install) separately and make sure it is accessed by global command.
```

Reply:
Ah so this isn't totally correct, `lint-staged` runs once just before the changes get committed to your branch, I've adjusted the wording to fit that so maybe take another look to make sure it seems reasonable 👍